themes/bootstrap3/js/truncate.js

```diff
@@ -184,6 +184,24 @@ VuFind.register('truncate', function Truncate() {
 
       container.addClass('truncate-done');
 
+      // Make hidden elements unfocusable
+      // - Create IntersectionObserver
+      const root = container.get(0);
+      const observer = new IntersectionObserver(
+        (entries) => {
+          entries.forEach((entry) => {
+            if (entry.intersectionRatio > 0) {
+              entry.target.removeAttribute("tabindex");
```

Comment:
Is this going to create problems if you truncate a region that contains explicit tab indexes? It seems to me like this will cause them all to get reset. Should we only remove the attribute if it equals -1? Or should we move any existing value to a data element so we can restore it exactly as it was?